pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -134,6 +205,18 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 	if newRc.Spec.Replicas == 0 {
 		newRc.Spec.Replicas = oldRc.Spec.Replicas
 	}
+	if cmdutil.GetFlagBool(cmd, "dry-run") {
```

Comment:
The others are grabbed in a bunch up here: https://github.com/GoogleCloudPlatform/kubernetes/pull/7268/files#diff-57b5708f59a1279bfe3c9e55adaef5a1R109 . Move this up?